seqr/views/utils/file_utils.py

```diff
@@ -86,25 +94,25 @@ def save_uploaded_file(request, process_records=None):
         raise ValueError("Received %s files instead of 1" % len(request.FILES))
 
     # parse file
-    stream = request.FILES.values()[0]
+    stream = list(request.FILES.values())[0]
```

Comment:
similar to above, I think using `next` to get the first value is better than converting it to a list and getting the first element

Reply:
Okay